lib/modules/datasource/docker/index.ts

```diff
@@ -453,6 +453,89 @@ export class DockerDatasource extends Datasource {
     }
   }
 
+  @cache({
+    namespace: 'datasource-docker-imageconfig',
+    key: (
+      registryHost: string,
+      dockerRepository: string,
+      configDigest: string
+    ) => `${registryHost}:${dockerRepository}@${configDigest}`,
+    ttlMinutes: 1440,
```

Comment:
I think this can be cached for at least 4 weeks. It should never change, but can maybe gone (404)
```suggestion
    ttlMinutes: 1440 * 14,
```